system-variables.md

```diff
@@ -295,9 +303,29 @@ This variable is an alias for `last_insert_id`.
 
 ### placement_checks <span class="version-mark">New in v5.3.0</span>
 
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> This TiDB variable is not applicable to TiDB Cloud.
+
+</CustomContent>
+
 - Scope: SESSION | GLOBAL
 - Default value: `ON`
+
+<CustomContent platform="tidb">
+
 - This variable controls whether DDL statements validate [Placement Rules in SQL](/placement-rules-in-sql.md).
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
```

Comment:
If this variable is not applicable to TiDB Cloud, do we still need to explain this variable to TiDB Cloud users?

Reply:
Since we cannot hide the section of this variable for now, it seems that there is no need to hide its explanation as well.